README.md

```diff
@@ -18,6 +18,30 @@ Requirements
 
 This is tested on and should run on any of Node.js `{8, 10}`.
 
+General Usage
+-------------
+
+While the capitalization and punctunation of the function names varies depending on the language, each language provides the following methods:
+
+| method | result |
+| --- | --- |
+| api(rootUrl, service, version, path) | `<rootUrl>/api/<service>/<version>/<path>` |
```

Comment:
I'd maybe use a hindley-milner notation? for clarity

#preferential

Reply:
*googles*

Reply:
It looks like that's a type system?  Can you give me an example of what you mean?